README.md

```diff
@@ -304,6 +304,7 @@ Here is the full list of actions and supported `params`
 | trace_retval         | isDateLogged, log_threshold_ms                |
 | counter              | isDateLogged, count_frequency                 |
 | avg_timing           | isDateLogged, window_length                   |
+| trace_login_config   | isDateLogged, entry_name                      |
```

Comment:
Please extend this documentation with the description of the new action (please refer to the javadoc of `javax.security.auth.login.Configuration` and show an example output).

Reply:
Added.